docs/DeveloperGuide.md

```diff
@@ -75,7 +75,6 @@ The `UI` component,
 ### Model component
 
 ![modelcomponent](diagrams/modelcomponent.png)
-
 The Model,
 * consists of ModuleList class that supports functions to execute module-related commands, 
   such as addmod, deletemod etc.
```

Comment:
i think the flow of the 1st addmod diagram is great and easy to understand

Reply:
2nd addmod diagram is clean and easy to understand as well, as compared to diagrams in 4.5 & 4.6, which tend to be a little bit more cluttered